docs/guide/localization/static.md

```diff
@@ -271,11 +233,48 @@ As a reminder, namespacing is primarily necessary for *installable modules* and
 
 ## Localizing the Apostrophe user interface
 
-The Apostrophe user interface contains many registered strings, currently localized to English. We will be working to provide more localization files, but if you are interested in adding l10n files to your project for the UI, you are welcome to do that.
+The Apostrophe user interface contains many registered strings, currently localized to English and a few other languages. We will be working to provide more localization files, but if you are interested in adding l10n files to your project for the UI, you are welcome to do that. Localizing text in your own **custom** admin UI modules is a separate topic which we'll address later.
+
+To localize Apostrophe's existing user interface strings, add JSON files for the locales to a project-level `modules/@apostrophecms/i18n/apostrophe` directory. You can also do this in any other module as long as you use an `i18n/apostrophe` subdirectory within that module, keeping these localizations separate from your project level strings. The core UI keys all use this namespace. Note that separately installed npm modules may use other namespaces, which you can see in the source code. You can create subdirectories for these as well.
```

Comment:
I don't think this is clear enough that `apostrophe` is the namespace. We hit them with some directory paths before mentioning a namespace but not saying what it is exactly. It depends on the reader already having internalized that the final directory name there is the namespace. We can be clearer about this.

> to a project-level `modules/@apostrophecms/i18n/apostrophe` directory. This adds them to the `apostrophe` namespace that the core UI uses. You can also do this...

Then the namespace sentence later can be removed.

Reply:
> ... keeping these localizations separate from your project level strings.

`project-level` should be hyphenated. It might also be good to reinforce namespacing here with "separate from your project-level namespace."